svn/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: svn
+  version: 1.9.3
+
+source:
+  fn: subversion-1.9.3.tar.bz2
+  url: http://www.trieuvan.com/apache/subversion/subversion-1.9.3.tar.bz2
+  sha1: 27e8df191c92095f48314a415194ec37c682cbcf
+  patches:
+    # - swig_regex.patch    [win and py>30]
+    #- swig_unicode_2.patch    [win and py>30]
+    #- swig_file_mode.patch    [win and py>30]
+    #- swig_init.patch    [win and py>30]
+    #- swig_gen_base.patch    [win and py>30]
+    #- swig_gen_win.patch    [win and py>30]
+    #- swig_external.patch    [win and py>30]
```

Comment:
Why not remove these comments?

Are these patchs present in the tree too? If that's the case, why not remove them too?